jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -10,13 +10,81 @@
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import (PluginTemplateMixin, ViewerSelect, LayerSelect,
                                         PlotOptionsSyncState)
+from jdaviz.core.user_api import PluginUserApi
 from jdaviz.core.tools import ICON_DIR
 
 __all__ = ['PlotOptions']
 
 
 @tray_registry('g-plot-options', label="Plot Options")
 class PlotOptions(PluginTemplateMixin):
+    """
+    The Plot Options Plugin gives access to per-viewer and per-layer options and enables
+    setting across multiple viewers/layers simultaneously.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``multiselect``:
+      whether ``viewer`` and ``layer`` should both be in multiselect mode.
+    * ``viewer`` (:class:`~jdaviz.core.template_mixin.ViewerSelect`):
+    * ``layer`` (:class:`~jdaviz.core.template_mixin.LayerSelect`):
+    * :meth:`select_all`
+    * ``layer_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)

```

Comment:
I have no idea if my suggestion is accurate or not, but this needs some explanation to differentiate it from other `visible` attributes. 

```suggestion
    * ``layer_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
      whether the selected layer is visible; this includes image and contours.
```

Reply:
I'll leave this unresolved for now until we make a decision on abstracting `layer_visible` away entirely, but if we stick with that approach, this will become irrelevant.